resources/styles/components/learning-guide.less

```diff
@@ -30,6 +30,7 @@
     margin: 0 auto;
 
     svg {
+      font-size:12px;
```

Comment:
looks like we are getting root rem from the .sans font defined in body in typography.less. that overwrites html 10px.  the svg needs 12px equivalent, so it might make more sense to have a global var for svg root rem